QuandlExcelAddinWindows/UI/UDF Builder/DateConditionSelection.xaml.cs

```diff
@@ -19,23 +19,39 @@ public DateConditionSelection(string identifier, FilterHelper filterHelper)
       
             Identifier = identifier;
             FilterHelper = filterHelper;
-            InitList();
+            dateSingleSelector();
+            
         }
 
         public FilterHelper FilterHelper { get; set; }
 
         public string Identifier { get; set; }
 
-        private void InitList()
+        public void dateRangeSelector()
         {
             List<Operator> listData = new List<Operator>();
-            listData.Add(new Operator { Value = "eq", Label = "=" });
             listData.Add(new Operator { Value = "lt", Label = "<"});
             listData.Add(new Operator { Value = "lte", Label = "<=" });
             listData.Add(new Operator { Value = "gt", Label = ">" });
             listData.Add(new Operator { Value = "gte", Label = ">=" });
-            ConditionBox.SelectedValue = "eq";
+            setComboboxList(listData, "lt");
+        }
+
+        public void dateSingleSelector()
+        {
+            List<Operator> listData = new List<Operator>();
+            listData.Add(new Operator { Value = "eq", Label = "=" });
+            listData.Add(new Operator { Value = "lt", Label = "<" });
+            listData.Add(new Operator { Value = "lte", Label = "<=" });
+            listData.Add(new Operator { Value = "gt", Label = ">" });
+            listData.Add(new Operator { Value = "gte", Label = ">=" });
```

Comment:
for method dateRangeSelector() and dateSingleSelector(), when create listData, could you remove the duplicated code?